docs_src/pipelinesetup.md

```diff
@@ -1,54 +1,50 @@
-# Pipeline Setup
+# Set up Pipeline
 
-## Step 1: Clone the repository
+### Step 1: Clone the repository
 
 ```
 git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
 ```
 
-## Step 2: Build Benchmark Containers
+### Step 2: Build the benchmark containers
 
-For benchmarking supported use cases you will need to build the benchmark containers
+For use cases that support benchmarking, build the benchmark containers:
 
 ```bash
 make build-all
 ```
 
-## Step 3: Download Models Manually (Optional)
+### Step 3: Download the models manually (Optional)
 
-Model downloader script is automatically called as part of docker-run.sh.  User can also download the models manually using the model downloader script shown as follows:
+When `docker-run.sh` the Model downloader script is automatically called. You can also download the models manually using the model downloader script:
 
 ```bash
 sh modelDownload.sh
 ```
 
-!!! note
-    This may take a while as there are several models to be downloaded for the first time.
+**_Note:_**  This step might take a while as the script will download several models for the first time.
 
-!!! note
-    To manually download models you can follow links provided in the [Model List](../configs/models/2022/models.list.yml)
+**_Note:_**  To manually download the models, see the links in [Model List](../configs/models/2022/models.list.yml).
```

Comment:
The step instructs to run a command to manually download the models. Then, what does the second note mean? This is a bit confusing.

Reply:
This should read "All models used can be found in the [Model List](../configs/models/2022/models.list.yml)"

Reply:
Are these the pre-trained models? Can it be "The pre-trained models are listed in Model List"?

Reply:
Yes that would make sense